Source/Tests/LinuxSyscalls/FileManagement.cpp

```diff
@@ -313,6 +278,44 @@ FileManager::FileManager(FEXCore::Context::Context *ctx)
     }
   }
 
+  // Now that we loaded the thunks object, walk through and ensure dependencies are enabled as well.
+  for (auto const &DBObject : ThunkDB) {
+    if (DBObject.second.Enabled) {
```

Comment:
To keep indentation levels low:
```suggestion
    if (!DBObject.second.Enabled) {
        continue;
    }
```

Reply:
Resolved.